plugins/flashrom/fu-flashrom-internal-device.h

```diff
@@ -0,0 +1,14 @@
+/* Copyright 2021 The Chromium OS Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
```

Comment:
BSD?! Not LGPLv2+?

Reply:
I'm not sure what are the implications here but that is the standard Chromium OS license.

Reply:
Looking at the cros_ec plugin files it seems it is fine to use LGPL... I just updated.